docs/user/build-customization.rst

```diff
@@ -104,6 +105,43 @@ To avoid this, it's possible to unshallow the clone done by Read the Docs:
          - git fetch --unshallow
 
 
+Skip build based on a condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Skipping builds
~~~~~~~~~~~~~~~
```

Reply:
While you are at it (SORRY!!!) - what about the automatic skipping, i.e. auto-canceling? https://blog.readthedocs.com/cancel-old-builds/

Reply:
I mean, in this subsubsection or an adjacent section, you could quickly mention that builds are automatically skipped if canceled and how that works?

Reply:
I only added a quick example for now. I agree that we should document these two things in a different place. For now, I put this "feature" in this particular page because it makes sense under "Build customization" since it requires the user to change the normal flow.

However, as you may thought already, a specific feature page mentioning both of them would probably be a good addition in another PR.

Reply:
In fact, I think a good place for this content is https://docs.readthedocs.io/en/stable/builds.html

Reply:
Moving it to the "Builds" page also sounds good, although a bit of how-to and explanation is still mixed.

I don't think you should feel blocked by the location of this content -- if you can develop the content, then we can move it later.

Reply:
I'm saying the code examples (how to) should live here, in `build-customization`. However, the introduction and description of the feature (explanation) should live in the `builds` page.

Reply:
I opened this PR that addresses these changes: https://github.com/readthedocs/readthedocs.org/pull/9717. Let's continue talking there, but I think it's 💯